components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/action/PreIssueAccessTokenResponseProcessor.java

```diff
@@ -71,8 +79,9 @@ public ActionType getSupportedActionType() {
     }
 
     @Override
-    public ActionExecutionStatus processSuccessResponse(Map<String, Object> eventContext, Event event,
-                                                        ActionInvocationSuccessResponse actionInvocationSuccessResponse)
+    public ActionExecutionStatus<Success> processSuccessResponse(Map<String, Object> eventContext, Event event,
+                                                                 ActionInvocationSuccessResponse
```

Comment:
Bring 83 and 84 lines into single line

Reply:
No this cannot be done as the line length exceeds